packages/api/tests/api/health.test.ts

```diff
@@ -3,7 +3,7 @@ import { testHandler } from '../testUtils/testHandler';
 
 describe('health', () => {
   it('returns ok true', async () => {
-    const { body } = await testHandler(health).expect(200);
+    const { body } = await testHandler(health).get('/').expect(200);
```

Comment:
Thanks for catching this! For the future, it'd be great if you could open a separate PR for things that aren't related, that way we can just merge immediately instead of bundling it as part of another review for another feature 😄

Reply:
Got it!

Reply:
Yup, that's what happened.